PyPoE/cli/exporter/wiki/parsers/item.py

```diff
@@ -3281,6 +3321,23 @@ def _export(self, parsed_args, items):
 
         return r
 
+    def _print_item_rowid(self, parsed_args, base_item_type):
+        #Don't print anything if not running in the rowid mode.
+        if (not 'start' in vars(parsed_args).keys()) or (not 'end' in vars(parsed_args).keys()):
```

Comment:
Same comment as https://github.com/Project-Path-of-Exile-Wiki/PyPoE/pull/37#discussion_r744102527